src/widgets/events/ui/events.tsx

```diff
@@ -1,33 +1,19 @@
 import cn from 'classnames';
 import classNames from 'classnames/bind';
 import dayjs from 'dayjs';
-import { Event, EventCard } from '@/entities/event';
+import { EventCard } from '@/entities/event';
 import photo3 from '@/shared/assets/photo-3.webp';
-import { getActualData } from '@/shared/helpers/getActualData';
 import { Image } from '@/shared/ui/image';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { SectionLabel } from '@/shared/ui/section-label';
 import { WidgetTitle } from '@/shared/ui/widget-title';
-import { events } from 'data';
+import { actualEvents, nearestEvents, rsLifetime } from '@/widgets/events/constants';
 
 import styles from './events.module.scss';
 
 const cx = classNames.bind(styles);
 
-const displayedCardsQuantity = 2;
-
-const actualEvents: Event[] = getActualData({
-  data: events,
-  staleAfter: 3,
-});
-
-const nearestEvents = actualEvents.slice(0, displayedCardsQuantity);
-
-const Stub = <Image src={photo3} alt="Community event" className={cx('event-img')} />;
-
 export const Events = () => {
-  const rsLifetime = dayjs().diff('2013', 'year');
-
   return (
     <article id="events" className={cn(cx('events'), 'container')}>
```

Comment:
```suggestion
    <article id="events" className={cx('container')}>
```
and delete the unnecessary class in `scss`

Reply:
done